data/vibe/data/json.d

```diff
@@ -2009,6 +2009,25 @@ struct JsonStringSerializer(R, bool pretty = false)
 			m_range.put('\n');
 			foreach (i; 0 .. m_level) m_range.put('\t');
 		}
+
+		private void sink (scope const(char)[] data)
+		{
+			static if (isOutputRange!(R, char))
+			{
+				static if (__traits(compiles, m_range ~= data))
```

Comment:
This needs to perform `jsonEscape`, or is the idea is that the object is coupled with the output serialization format and can write arbitrary data?